### ISSUE TITLE
List stream only streaming tweets from followed accounts

### ISSUE THREAD

**Issue by shayanjm** [DTVD/rainbowstream#164]
When switching to a public list (i.e: switch list @pmarca/economics) the only tweets that are streamed are from accounts I currently follow. The expected behavior would be to stream all tweets coming from accounts in the list regardless of whether or not I follow them.

**Comment by stianlagstad** [comment]
I agree with this: The expected behavior would be to stream all tweets coming from accounts in the list regardless of whether or not I follow them.
